docs/local-dind-tutorial.md

```diff
@@ -48,8 +48,7 @@ NAME                             AGE
 tidbclusters.pingcap.com         1m
 
 $ # Install TiDB Operator into Kubernetes
-$ helm install charts/tidb-operator --name=tidb-operator --namespace=tidb-admin
-
+$ helm install charts/tidb-operator --name=tidb-operator --namespace=tidb-admin --set scheduler.kubeSchedulerImageName=mirantis/hypokube --set scheduler.kubeSchedulerImageTag=final
```

Comment:
Why not change it in `values.yaml` directly?

Reply:
This is a tutorial, it's easier for users to copy/paste.